decorator.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'nameable'
+class Decorator < Nameable
+  def initialize(nameable)
```

Comment:
- [ ] Kindly fix this error
![image](https://user-images.githubusercontent.com/24703171/233054773-0c7d7570-2ef5-418d-a7b7-0cf8931d00bc.png)
I get this error when I run the `test.rb` file. This error happens because you use a property called `nameable` inside the method `correct_name` in the `CapitalizeDecorator` class that is not defined. Also, you will get the same error in the `TrimmerDecorator` class as you use the same property. You can fix this error by defining an `attr_accessor` for the `@nameable` property inside the `Decorator`  class.